PyISY/Nodes/__init__.py

```diff
@@ -187,8 +187,22 @@ def parse(self, xml):
                                 dimmable = '%' in \
                                     nprop[0].attributes['uom'].value
                                 nval = int(nval.replace(' ', '0'))
+                            # Get the device notes, currently only the Spoken property is saved.
+                            notes_xml = self.parent.conn.getNodeNotes(nid)
+                            spoken = None
+                            if notes_xml is not None:
+                                try:
+                                    notesdom = minidom.parseString(notes_xml)
+                                except:
```

Comment:
It would be best to include the types of errors to handle instead of handling them all. I know that I use vague excepts a lot in this library, but I originally wrote a log of this as a way to learn Python. Moral of the story, I didn't always follow best practices and set the best example. Live and learn.